doc/classes/SpriteFrames.xml

```diff
@@ -39,6 +39,14 @@
 				Removes all animations. An empty [code]default[/code] animation will be created.
 			</description>
 		</method>
+		<method name="duplicate_animation">
+			<return type="void" />
+			<param index="0" name="anim_from" type="StringName" />
+			<param index="1" name="anim_to" type="StringName" />
+			<description>
+				Add a new [param anim_to] animation and duplicate all frames from [param anim_from] to [param anim_to].
```

Comment:
It's not just the frames, it's everything about it (frames, speed, and looping).
Not sure about this suggestion myself but it _may_ be an improvement.
```suggestion
				Duplicates the animation [param anim_from] onto a new animation named [param anim_to]. Fails if [param anim_to] already exists.
```

Reply:
Yes, you're right, this way is clearer.
I noticed that other method's description didn't include explanation for failures (such as `add_animation`). Do we need to keep the documentation consistent?

Reply:
> I noticed that other method's description didn't include explanation for failures
As per usual. I think it's okay to keep the other descriptions as they are, but worth revisiting another time. SpriteFrames is quite "old".